src/Hydrators/ProductHydrator.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace RobotStores\Hydrators;
+
+class ProductHydrator
+{
+
+    static public function getFilteredProducts(): array
```

Comment:
at this point, there is no filter. i would say drop this method and use the getAllProducts method at this point. this method litterally does nothing